bot/seasons/season.py

```diff
@@ -38,12 +38,19 @@ def get_season(bot, season_name: str = None, date: datetime.date = None):
 
     seasons = get_seasons()
 
-    # If there's a season name, we can just grab the correct class.
+    # Use season override if season name not provided
+    if not season_name and Client.season_override:
+        log.debug(f"Season override found: {Client.season_override}")
+        season_name = Client.season_override
+
+    # Check if a valid season name was provided
     if season_name:
         season_name = season_name.lower()
         if season_name not in seasons:
             season_name = 'evergreen'
 
+    # If there's a season name, we can just grab the correct class.
+    if season_name:
```

Comment:
Can this not just be moved under the previous if statement?

Reply:
It can, again a remnant of some deleted logic. Thanks.